matrix-sdk/src/main/java/org/matrix/androidsdk/crypto/MXCrypto.java

```diff
@@ -2427,22 +2428,6 @@ public void onSuccess(MXUsersDevicesMap<MXDeviceInfo> devicesMap) {
      * @param callback the asynchronous callback.
      */
     public void setGlobalBlacklistUnverifiedDevices(final boolean block, final ApiCallback<Void> callback) {
-        final String userId = mSession.getMyUserId();
-        final List<String> userRoomIds = new ArrayList<>();
-
-        Collection<Room> rooms = mSession.getDataHandler().getStore().getRooms();
-
-        for (Room room : rooms) {
-            if (room.isEncrypted()) {
-                RoomMember roomMember = room.getMember(userId);
-
-                // test if the user joins the room
-                if ((null != roomMember) && TextUtils.equals(roomMember.membership, RoomMember.MEMBERSHIP_JOIN)) {
-                    userRoomIds.add(room.getRoomId());
-                }
-            }
-        }
-
```

Comment:
Ok to remove this dead code. Missing cleanup from 444644570ac6b424b700d38879a839247ba80f4f